### DISCUSSION TITLE
Objective Fun -0.0000000

### DISCUSSION THREAD

**Discussion by dzianismr** [nlmixr2/nlmixr2#196] [Q&A]
I am setting up nlmixr run for fairly large model. The closest example is mavoglurant pbpk model (https://nlmixr2.org/articles/mavoglurant.html). The problem I am strugling with is the following: nlmixr runs with focei and similar methods show constant value of objective function (in case of "fo" OFV = -0.000). The dataset has a minimum set of columns:   "ID"   "TIME" "CMT"  "EVID" "DV"   "AMT"  "MDV" and when I run simulation (nlmixr_model  %>%  rxSolve( events = nlmixr_dataset ) ) prior to nlmixr optimisation the overlay betwen the observed data and model output is very reasonable.  Model output is sensetive to model parameters. Changing outerOpt to "bobyqa" and/or type of error model does not help.

Any thoughts/tips?

**Comment by mattfidler** [comment]
First, I would try to fit without any random effects; That way you can get some initial estimates that make sense.  If you have the development version of `nlmixr2est` there are a new estimation methods you can try (not just `focei` family);  You can also use `gnlmm`, `optim`, `nlminb` and more.  In those methods, the full gradient would be calculated symbolically and it may perform better in your problem to get the best initial estimates.

Once you have these good estimates add a between subject variability starting at those estimates and see how it performs.

**Reply by dzianismr** [reply]
Thanks @mattfidler. It does work w/o random effects. Will check other leads.
